iou-flow/src/main/resources/migration/iou.changelog-master.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
Testing this out, I accidentally only promoted and staged only the `iou-contract` JAR to a Corda node and it failed 
bc this file was not in the classpath bc its being bundled into the `iou-flow` JAR instead.

If I ensure to promote / stage both CordApps at the same time, the node starts up w/ no issue. So it is unclear to me if the two CordApps should be so dependent on one another, or if these migration files simply need to be moved underneath the `iou-contract` `src` folder instead.

Reply:
Chatting w/ @jimthematrix we determined the coupling of `iou-contract` to `iou-flow` is fine / expected. So this should be good to ship as I've successfully deployed these CordApps to all my Corda nodes in a network.

Reply:
thanks @hfuss for confirming the successful deployment of the cordapps with the enhancement. as such I'll approve and merge the PR